src/main/java/uk/gov/hmcts/reform/sscscorbackend/service/DecisionExtractor.java

```diff
@@ -0,0 +1,103 @@
+package uk.gov.hmcts.reform.sscscorbackend.service;
+
+import static java.lang.Integer.parseInt;
+import static java.time.LocalDate.of;
+import static java.util.Arrays.asList;
+import static java.util.stream.Collectors.toList;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.text.StringEscapeUtils;
+import org.json.JSONObject;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.sscscorbackend.domain.*;
+
+@Component
+public class DecisionExtractor {
+
+    private static final Map<String, String> activityMappings = new HashMap<>();
+    private static final String AWARD = "award";
+
+    static {
+        activityMappings.put("preparingFood", "dailyLivingPreparingFood");
+        activityMappings.put("takingNutrition", "dailyLivingTakingNutrition");
+        activityMappings.put("managingTherapy", "dailyLivingManagingTherapy");
+        activityMappings.put("washingBathing", "dailyLivingWashingBathing");
+        activityMappings.put("managingToilet", "dailyLivingManagingToilet");
+        activityMappings.put("dressingUndressing", "dailyLivingDressingUndressing");
+        activityMappings.put("communicatingVerbally", "dailyLivingCommunicatingVerbally");
+        activityMappings.put("readingAndUnderstanding", "dailyLivingReadingSigns");
+        activityMappings.put("engagingWithOtherPeople", "dailyLivingEngangingPeople");
+        activityMappings.put("makingBudgetingDecisions", "dailyLivingMakingBudgetDecisions");
+        activityMappings.put("planningFollowingJourneys", "MobilityPlanningJourneys");
+        activityMappings.put("movingAround", "MobilityMovingAround");
+    }
+
+    private static final List<String> dailyLivingActivities = asList(
+            "preparingFood", "takingNutrition", "managingTherapy", "washingBathing", "managingToilet",
+            "dressingUndressing", "communicatingVerbally", "readingAndUnderstanding", "engagingWithOtherPeople",
+            "makingBudgetingDecisions");
+    private static final List<String> mobilityActivities = asList("planningFollowingJourneys", "movingAround");
+
+    public Decision extract(long caseId, CohDecision decision, CohDecisionReply appellantReply) {
+        String unescapeJson = StringEscapeUtils.unescapeJava(decision.getDecisionText());
+
+        JSONObject jsonObject = new JSONObject(unescapeJson);
+        JSONObject decisionJson = jsonObject.getJSONObject("decisions_SSCS_benefit_" + caseId);
+
+        String startDateString = getDate(decisionJson, "Start");
+        String endDateString = getDate(decisionJson, "End");
```

Comment:
So infinite looks like this
,\"awardStartDateDay\":\"12\",\"awardStartDateMonth\":\"12\",\"awardStartDateYear\":\"2018\",\"approveDraftConsent\":\"indefinite\",\"awardEndDateDay\":null,\"awardEndDateMonth\":null,\"awardEndDateYear\":null,